pkg/report/app_instange_tags.go

```diff
@@ -0,0 +1,105 @@
+package report
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated-sdk/pkg/store"
+	"github.com/replicatedhq/replicated-sdk/pkg/util"
+	"k8s.io/client-go/kubernetes"
+)
+
+func SendAppInstanceTags(clientset kubernetes.Interface, sdkStore store.Store, data map[string]string) error {
+	if util.IsAirgap() {
+		return SendAirgapAppInstanceTags(clientset, sdkStore, data)
+	}
+	return SendOnlineAppInstanceTags(sdkStore, data)
+}
+
+func SendAirgapAppInstanceTags(clientset kubernetes.Interface, sdkStore store.Store, data map[string]string) error {
+	report := &AppInstanceTagsReport{
+		Events: []AppInstanceTagsReportEvent{
+			{
+				ReportedAt: time.Now().UTC().UnixMilli(),
+				LicenseID:  sdkStore.GetLicense().Spec.LicenseID,
+				InstanceID: sdkStore.GetAppID(),
+				Data:       data,
+			},
+		},
+	}
+
+	if err := AppendReport(clientset, sdkStore.GetNamespace(), report); err != nil {
```

Comment:
Is this treated as reporting data? I would expect this data to not be historical and be sent with instance status on every request

Reply:
That is a good point!  Do you ever see a use-case where we might historical data? I do not.